.github/workflows/hypersdk-ci.yml

```diff
@@ -147,13 +152,12 @@ jobs:
             go.sum
             examples/morpheusvm/go.sum
       - name: Run e2e tests
-        working-directory: ./examples/morpheusvm
-        shell: bash
-        run: scripts/run.sh
-        env:
-          MODE: 'test'
-      - name: Upload tmpnet network dir
-        uses: ava-labs/avalanchego/.github/actions/upload-tmpnet-artifact@v1-actions
-        if: always()
+        uses: ava-labs/avalanchego/.github/actions/run-monitored-tmpnet-cmd@8551e1d96221c8a5c2c034c2bad0cd571610ad35
```

Comment:
Latest commit of AvalancheGo we updated to https://github.com/ava-labs/avalanchego/commit/8551e1d96221c8a5c2c034c2bad0cd571610ad35